0001-typedb/02-dev/02-schema.md

```diff
@@ -0,0 +1,911 @@
+
+After a successful commit, all the types can be seen in the Types panel in the bottom left part of the Studio window. 
+In addition, the following query can be run in a new tab in **Schema** / **Read** mode to see the schema visualized 
+as a graph (as shown below).
+
+<!-- test-ignore -->
+```typeql
+match $s sub thing;
+```
+
+![Example 1](../../images/schema/example-1.png)
+
+<div class="note">
+[Warning]
+The `thing` base type will be deprecated in TypeDB version 3.0. 
+To produce the same result as the above example, use the following query:
+
+<!-- test-ignore -->
+```typeql
+match $s sub $t; { $t type entity; } or { $t type relation; } or { $t type attribute; };
+```
+</div>
+
+In the following sections, you can find more detailed information on different schema definition queries and 
+possibilities.
+
+### Define entity types
+
+#### Overview
+
+Entity types are defined independently of other types, but may subtype other entity types.
+
+Optionally, entity types can:
+
+* be abstract.
+* own attribute types.
+* play roles defined in relation types.
+
+#### Syntax
+
+Entity types are defined in TypeQL with the following pattern:
+
+<!-- test-ignore -->
+```typeql
+<name> sub (entity | <entity type label>) [(, abstract)]
+[(, owns <attribute type name)...]
+[(, plays <relation type name>:<role>)...];
+```
+
+#### Examples
+
+##### Basic
+
+The following `define` query creates a new entity type, `object`, by subtyping the `entity` base type.
+
+<!-- test-ignore -->
+```typeql
+define object sub entity;
+```
+
+##### Abstract
+
+Optionally, entity types can be defined as `abstract`. An abstract entity type can’t be instantiated and must be 
+subtyped in order to create entities. It exists only so other entity types can inherit the attribute types it owns 
+and the roles it plays.
+
+<!-- test-ignore -->
+```typeql
+define object sub entity, abstract;
+```
+
+##### Owns an attribute
+
+To define a new entity type that owns one or more attribute types, use the `owns` keyword followed by the name of the 
+attribute type. The attribute types are appended to the entity type definition with commas. Note, attribute types must 
+be defined before or concurrently (in the same transaction) with its owner(s). You can add owners later but you can't 
+own non-existent attribute type.
+
+<!-- test-ignore -->
+```typeql
+define
+
+object-type sub attribute, value string;
+object sub entity, abstract, owns object-type;
+```
+
+###### Cardinality
+
+By default, an entity can have multiple attributes of the same type: zero, one, or many.
+
+By having multiple attributes of the same type we're effectively creating a 
+[multivalued attribute](04-write.md#multivalued-attributes) (i.e. having an attribute type instantiated with multiple 
+values).
+
+Use the `@key` keyword to limit the cardinality to exactly **one** and add **uniqueness** constraint. 
+See example in [Key attribute](#define-attribute-types) section.
```

Comment:
Yes and no. The very first sentence is the most important one here. The rest is just additional info. When we will add the cardinality option it will be added here, but it's not the only goal here. Reminding users that we can have 0, 1 or many attributes of the same type owned is quite important.